ostd/src/mm/io.rs

```diff
@@ -391,6 +391,35 @@ impl<'a> VmReader<'a, KernelSpace> {
         self.read(&mut writer);
         Ok(val)
     }
+
+    /// Reads a value of the `Pod` type using one non-tearing memory load.
+    ///
+    /// If the length of the `Pod` type exceeds `self.remain()`, this method will return `Err`.
+    ///
+    /// # Panics
+    ///
+    /// This method will not compile (due to failed constant assertions) if the `Pod` type is too
+    /// large for the current architecture and the operation must be tear into multiple memory
+    /// loads.
+    ///
+    /// This method will panic if the current position of the reader does not meet the alignment
+    /// requirements of type `T`.
+    pub fn read_val_no_tearing<T: Pod>(&mut self) -> Result<T> {
```

Comment:
`read_once` seems to be a better name

Reply:
I follow the term as it is used when the Rust team discusses the concept. For example, see https://rust-lang.zulipchat.com/#narrow/stream/136281-t-opsem/topic/Non-tearing.20volatile.20accesses.

Reply:
Tearing means split a read/write into multiple reads/writes. So non-tearing means the same as once, but the latter is shorter. So I am ok with "non-tearing", but prefer "once"

Reply:
OK. I also like shorter names. So I'll change it accordingly soon.